### ISSUE TITLE
Defollow not working

### ISSUE THREAD

**Issue by ivanga** [social-manager-tools/socialmanagertools-igbot#175]
### Versions

- **Bot Version:** v0.10.0
- **Node Version:** v8 (Bot require >= latest lts version)
- **Operating System:** Ubuntu 18
- **Browser:** chromium
- **Mode:** fdf_classic 

### Expected Behavior

Defollow mode not working, caused whitelist work wrong
Whitelist is filled with all the accounts for which the bot has just subscribed.

### Actual Behavior

[INFO] fdf_classic: classic

[INFO] fdf_classic: fdf_get_urlpic

[INFO] fdf_classic: current photo url https://www.instagram.com/p/By3QFHGiTIV/
[INFO] fdf_classic: try follow
[INFO] fdf_classic: username nicotackle

[DEBUG] fdf_classic: whitelist 

[INFO] fdf_classic: try follow
[INFO] fdf_classic: username _boginya_lubvi_
[INFO] fdf_classic: users already followed count 1
[DEBUG] fdf_classic: whitelist nicotackle

### Steps to Reproduce

1. Just set "fdfmode_classic" as mode.

### config.js (remember to remove password)

```
module.exports = {
        // InstagramBot.js Configs
        "debug": true,
        "login": true,
        "ui": false, // only for social-manager-tools

        // Instagram Account
        "instagram_username": "", // without @
        "instagram_password": "",
        "instagram_hashtag": [], // without #, recommended more than 20
        "instagram_pin": "email", // method to receive pin (email or sms)

        // BOT Configs
        // [WORK] likemode_classic          - select random hashtag from config list and like 1 random photo (of last 20) | 400-600 like/day.
        // [WORK] likemode_realistic        - select random hashtag from config list, like fast 10-12 photo and sleep 15-20min. Sleep at night
        // [WORK] likemode_superlike        - select random hashtag from config list and like 3 random photo of same user | 400-600 like/day.
        // [WORK] likemode_competitor_users - select account, select random followers, like 10-12 photo and sleep 15-20min.
        // [WORK] comment_mode              - leave a comment under the photo via hashtag.
        // [WORK] fdfmode_classic           - follow 30 users, and defollow the first followed at 31 follow (in loop). This method is not detected from socialblade. ~1h | 300/300 follow-defollow/day.
        "bot_mode": "fdfmode_classic",
        "bot_likeday_min": 800,      // work in all likemode_*
        "bot_likeday_max": 900,      // work in all likemode_*. In 2018 limit instagram is 1000/day
        "bot_sleep_night": false,    // if you need stop bot in night
        "bot_start_sleep": "7:00",   // sleep from 00:00 to 7:00 am, work in likemode_realistic, comment_mode, likemode_superlike and likemode_competitor_users
        "bot_superlike_n": 3,        // superlike number of likes for user
        "bot_followday": 300,        // number of follow-defollow user/day, work in fdfmode_classic. In 2018 limit instagram is 500/day
        "bot_followrotate": 30,      // follow 30 users, and defollow the first followed at 31 follow (in loop), work in fdfmode_classic | If is 0 defollow all followed previously by bot
        "bot_userwhitelist": [],   // usernames without @ | don't defollow this users on follow-defollow mode, work in fdfmode_classic
        "comment_mode": {
                "comments": {
                        "type": "array",
                        "source": ["comment1", "comment2", "comment3"],
                }
        },
        "likemode_competitor_users": {
                "account": "user_nickname" // without @
        },

        // Puppeteer Configs
        "chrome_headless": true,
        "chrome_options": ["--disable-gpu", "--no-sandbox", "--window-size=1920x1080"],
        "executable_path": "", // example for Mac OS: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome

        // DATABASES
        "fdfdatabase_path": "./databases/fdf.db", // databases with username of followed users with follodefollow mode
        "logdb_path": "./databases/logs.db",      // logs in sqllite database for better report


     "logdb_path": "./databases/logs.db",      // logs in sqllite database for better report

        // LOG
        "pin_path": "./loginpin.txt",
        "log_path": "./logs/debug.log",
        "logerr_path": "./logs/errors.log",
        "screenshot_path": "./logs/screenshot/",
        "log": {
                "drivers": ["console"], // slack or console
                "screenshot": false,  // disable or enable screenshot in logs folder
                "channels": {
                        "console": "",
                        "slack": {
                                "webhook": ""
                        }
                }
        }
};




```

### Screenshots (Optional)

![Screenshot_1](https://user-images.githubusercontent.com/7168976/59717953-92b15680-9208-11e9-8e97-61a370edeaf4.png)

**Comment by ptkdev** [comment]
v0.10 is a beta release, at moment followdefollow in v0.10 is broken. Thanks for feedback!

**Comment by ptkdev** [comment]
These days, instagram servers in Europe are down, it's difficult to test and fix the bugs. For example, today hashtags have "no photos". Stay tuned...

**Comment by ptkdev** [comment]
**Fixed** in Social Manager Tools v0.6.6 and IGBot Library v0.9.18. 
_Remember_: update and use latest chrome version (v75)

[![](https://img.shields.io/badge/donate-paypal-005EA6.svg)](http://paypal.ptkdev.io) [![](https://img.shields.io/badge/donate-patreon-F87668.svg)](http://patreon.ptkdev.io) [![](https://img.shields.io/badge/donate-opencollective-5DA4F9.svg)](http://opencollective.ptkdev.io) [![](https://img.shields.io/badge/buy%20me-coffee-4B788C.svg)](http://coffee.ptkdev.io) [![](https://img.shields.io/badge/help-support@ptkdev.io-fbbc05.svg)](mailto:support@ptkdev.io)
